### PR TITLE
Updated suggest project

### PR DESCRIPTION
**Pull request by BrunoLagoa**
Hello, some contributions to the project. 🤗

- Add .gitignore in project
- Removed css in html
- Add css in project
- Updated Readme

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -29,6 +29,11 @@ body {
   --slider-height: calc(var(--switch-height) - var(--switch-padding));
   --slider-on: calc(var(--switch-height) - var(--switch-padding));
 }
+
+.power-switch {
+  margin: 30px;
```
Comment (immattdavison) [inline style.css:34]: this margin does not match the original

File: style.css [hunk 2]
```diff
@@ -50,31 +55,31 @@ body {
   right: 0;
   bottom: 0;
   background-color: red;
-  -webkit-transition: .4s;
-  transition: .4s;
+  -webkit-transition: 0.4s;
+  transition: 0.4s;
```
Comment (immattdavison) [inline style.css:59]: unnecessary.

File: style.css [hunk 3]
```diff
@@ -50,31 +55,31 @@ body {
...
 }
 
 .slider:before {
-  content: "";
+  content: '';
```
Comment (immattdavison) [inline style.css:63]: change for the sake of change. unnecessary.

File: style.css [hunk 4]
```diff
@@ -50,31 +55,31 @@ body {
...
   position: absolute;
   height: var(--slider-height);
   width: var(--slider-height);
   left: calc(var(--switch-padding) / 2);
   bottom: calc(var(--switch-padding) / 2);
   background-color: white;
-  -webkit-transition: .4s;
-  transition: .4s;
+  -webkit-transition: 0.4s;
+  transition: 0.4s;
```
Comment (immattdavison) [inline style.css:71]: unnecessary.

File: style.css [hunk 5]
```diff
@@ -50,31 +55,31 @@ body {
...
+  content: '';
   position: absolute;
   height: var(--slider-height);
   width: var(--slider-height);
   left: calc(var(--switch-padding) / 2);
   bottom: calc(var(--switch-padding) / 2);
   background-color: white;
-  -webkit-transition: .4s;
-  transition: .4s;
+  -webkit-transition: 0.4s;
```
Comment (immattdavison) [inline style.css:70]: unnecessary.

File: style.css [hunk 6]
```diff
@@ -50,31 +55,31 @@ body {
...
+  transition: 0.4s;
 }
 
-input:checked+.slider {
+input:checked + .slider {
```
Comment (immattdavison) [inline style.css:74]: unnecessary.

File: style.css [hunk 7]
```diff
@@ -50,31 +55,31 @@ body {
...
   background-color: green;
 }
 
-input:focus+.slider {
-  box-shadow: 0 0 1px  green;
+input:focus + .slider {
```
Comment (immattdavison) [inline style.css:78]: unnecessary.

File: style.css [hunk 8]
```diff
@@ -50,31 +55,31 @@ body {
...
+  box-shadow: 0 0 1px green;
```
Comment (immattdavison) [inline style.css:79]: unnecessary.

File: style.css [hunk 9]
```diff
@@ -50,31 +55,31 @@ body {
...
 }
 
-input:checked+.slider:before {
+input:checked + .slider:before {
```
Comment (immattdavison) [inline style.css:82]: unnecessary

File: README.md [hunk 10]
```diff
@@ -1,5 +1,10 @@
 <h1 align="center">NoMoreDomains</h1>
 
+<div align="center">
+    <img alt="Desktop" title="#Desktop" src="./assets/mock.jpg" width="680px" />
```
Comment (immattdavison) [inline README.md:4]: this mock-up is not what we need, it does not show an accurate representation of this tool.

File: .gitignore [hunk 11]
```diff
@@ -0,0 +1,2 @@
+.DS_Store
```
Comment (immattdavison) [inline .gitignore:1]: these are being added for the sake of being added it would seem.

File: .gitignore [hunk 12]
```diff
@@ -0,0 +1,2 @@
...
+.vscode
```
Comment (immattdavison) [inline .gitignore:2]: unnecessary.

File: popup.html [hunk 13]
```diff
@@ -1,32 +1,32 @@
...
-    <script src="blockpage.js" type="text/javascript"></script>
-    <script src="popup.js" type="text/javascript"></script>
-  </body>
-</html>
+  <title>No More Domains</title>
+</head>
+
+<body>
+  <div class="title-block">
+    <h1 style="margin-bottom: 0.2em">No More Domains</h1>
```
Comment (immattdavison) [inline popup.html:17]: why remove some inline css but not all?

File: popup.html [hunk 14]
```diff
@@ -1,32 +1,32 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
[the remark this answers is not in the document]
Comment (BrunoLagoa) [reply popup.html:3]: spam?

[the remark this answers is not in the document]
Comment (immattdavison) [reply popup.html:3]: > spam?

well that's what it looks like, yes?

File: style.css [hunk 15]
```diff
@@ -2,8 +2,8 @@ html,
 body {
   width: 420px;
   text-align: center;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
-    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen,
+    Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
```
[the remark this answers is not in the document]
Comment (BrunoLagoa) [reply style.css:6]: lint

File: style.css (hunk 1 above)
Comment (BrunoLagoa) [reply style.css:34]: <div class="power-switch" style="margin: 3px;">

File: style.css (hunk 9 above)
Comment (BrunoLagoa) [reply style.css:82]: space

File: popup.html (hunk 13 above)
Comment (BrunoLagoa) [reply popup.html:17]: <div class="power-switch" style="margin: 3px;">

File: .gitignore (hunk 12 above)
Comment (BrunoLagoa) [reply .gitignore:2]: Added to not enter commits

Comment (immattdavison) [reply .gitignore:2]: but these commits aren't there to be made!

**Comment by BrunoLagoa** [comment]
I understand not accepting the suggestions, I do not agree to classify it as spam.

**Comment by immattdavison** [comment]
> I understand not accepting the suggestions, I do not agree to classify it as spam.

It is spam, none of these changes make any difference to the way the project works/runs other than the mockup which makes it appear you have never used the project, are you aware of whether the project is a website/browser extension/mobile app?

**Comment by BrunoLagoa** [comment]
Hello, I would consider this PR as invalid, I understand that it did not sendito to you, as eslint formatted your code. It could be considered invalid and not spam. I would like to continue my contributions.

**Pull request closed without merging**